docs/src/Modal.doc.js

```diff
@@ -1,17 +1,26 @@
 // @flow strict
 import type { Node } from 'react';
 import PropTable from './components/PropTable.js';
-import Example from './components/Example.js';
 import PageHeader from './components/PageHeader.js';
-import Card from './components/Card.js';
+import MainSection from './components/MainSection.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
 card(
   <PageHeader
     name="Modal"
-    description="Modals are light temporary views; general purpose containers allowing for a multitude of design choices."
+    description="Modals display content that requires user interaction on a layer above the page content. Modals block the content underneath, preventing users from interacting with anything else besides the Modal.
```

Comment:
nit: the first sentence is a little hard to parse at first glance. Does the content require user interaction, or the Modal? Is the user interaction on a layer of the page content, or is the Modal?